apps/api/src/server/applications/application/file-folders/folders.validation.js

```diff
@@ -20,7 +20,7 @@ const validateExistingFolderBelongsToCase = async (value, { req }) => {
 	}
 };
 
-export const validateFolderId = composeMiddleware(
+export const validateOptionalFolderId = composeMiddleware(
```

Comment:
there is a fn that returns folders on a case, it can also work for the top level folders - but there is no folderid in that case, hence the optional validation.

Reply:
Ok, yes that is correct as to the functionality that we provide. However, we use separate endpoints for returning folder information for top level folders and sub folders. This same validator has been used for endpoints where `folderId` is required. Could you please check this over

Reply:
@mayya-bond-kc I have checked, you are correct - all the endpoints that called the validation have folderId as a required parameter.  I have therefore changed them all to the required version, and deleted the optional version as it is no longer used